test/test_automl/test_automl.py

```diff
@@ -62,6 +62,9 @@ def test_fit(dask_client):
     score = automl.score(X_test, Y_test)
     assert score > 0.8
     assert count_succeses(automl.cv_results_) > 0
+    assert includes_train_scores(automl.performance_over_time_.columns) is True
+    assert performance_over_time_is_plausible(automl.performance_over_time_) is True
+    assert performance_over_time_is_plausible(automl.performance_over_time_) is True
     assert automl._task == MULTICLASS_CLASSIFICATION
```

Comment:
* There is two of the same assertions, please remove one.
* The `is True` part isn't necessary but it causes no harm so it can be left if you wish but I would personally prefer without them.